be/src/storage/lake/lake_local_persistent_index.cpp

```diff
@@ -34,7 +34,7 @@ Status LakeLocalPersistentIndex::load_from_lake_tablet(TabletManager* tablet_mgr
     }
 
     std::unique_ptr<TabletLoader> loader =
-            std::make_unique<LakeLocalPersistentIndexTabletLoader>(tablet_mgr, metadata, base_version, builder);
+            std::make_unique<LakeLocalPersistentIndexTabletLoader>(tablet_mgr, metadata, base_version, builder, this);
```

Comment:
is there any way to ensure that the LakeLocalPersisentIndex lives longer than the TabletLoader, so the `this` parameter is always valid during the lifecycle of the loader?

Reply:
`loader` is constructed in `load_from_lake_tablet`. And `load_from_lake_tablet` is called by `LakeLocalPersistentIndex` when is loaded. `Loader` will be destructed in `load_from_lake_tablet`.